Template10 (Library)/Controls/HamburgerButtonInfo.cs

```diff
@@ -83,6 +79,14 @@ public bool IsEnabled
             DependencyProperty.Register(nameof(IsEnabled), typeof(bool),
                 typeof(HamburgerButtonInfo), new PropertyMetadata(true));
 
+        public string ToolTipText
+        {
+            get { return (string)GetValue(ToolTipTextProperty); }
+            set { SetValue(ToolTipTextProperty, value); }
+        }
+        public static readonly DependencyProperty ToolTipTextProperty =
+            DependencyProperty.Register("ToolTipText", typeof(string), typeof(HamburgerButtonInfo), new PropertyMetadata(null));
```

Comment:
This is just a style comment to use nameof(ToolTipText) instead of the literal "ToolTipText" which, of course, would break if the property name changes, though we would not get a compile error. Make sense? Just good practice, is all. I know the snippet did this.